src/api/base.ts

```diff
@@ -86,3 +87,15 @@ export abstract class Base {
     return this.request<T>(path, { ...options, method: 'DELETE' });
   }
 }
+
+function readCustomHeaders() {
```

Comment:
I'd prefer not to make the environment variable part of the SDK and instead have it only be used in the tests.

Reply:
We have to expose something then, otherwise the only way is to patch request or fetch function.

Reply:
The julia SDK has a way to generically pass in additional arguments to all the functions which they use for custom headers:

Julia: https://github.com/RelationalAI/rai-sdk-julia/blob/main/src/api.jl#L524

The Go SDK just mutates the http client when running tests: https://github.com/RelationalAI/rai-sdk-go/blob/3e6dfd815d04050e134375642d3c29eb6a66e391/rai/main_test.go

Reply:
It's more like in Go now.